cmd/project/project.go

```diff
@@ -3,18 +3,46 @@ package project
 import (
 	projectapi "github.com/CircleCI-Public/circleci-cli/api/project"
 	"github.com/CircleCI-Public/circleci-cli/cmd/validator"
+	"github.com/CircleCI-Public/circleci-cli/prompt"
 
 	"github.com/CircleCI-Public/circleci-cli/settings"
 	"github.com/spf13/cobra"
 )
 
+// UserInputReader displays a message and reads a user input value
+type UserInputReader interface {
+	ReadSecretString(msg string) (string, error)
+	AskConfirm(msg string) bool
+}
+
 type projectOpts struct {
 	client projectapi.ProjectClient
+	reader UserInputReader
+}
+
+// ProjectOption configures a command created by NewProjectCommand
+type ProjectOption interface {
```

Comment:
I referred https://pkg.go.dev/google.golang.org/grpc#CallOption to make this option.